### DISCUSSION TITLE
History

### DISCUSSION THREAD

**Discussion by AbbasSrour** [Morganamilo/paru#599] [Q&A]
Does paru have a history functionality to check what packages where installed or updated every transaction. If not it would be a great feature to have if you forgot what packages you upgraded and ended up with a brocken system. I know that living on the bleeding edge entitles you to be more careful before upgrading, and knowing what you are upgrading but humans being humans always fuck up.

**Comment by Morganamilo** [comment]
The pacman log should log this.
